lib/mayaUsd/commands/baseExportCommand.cpp

```diff
@@ -205,7 +205,7 @@ MStatus MayaUSDExportCommand::doIt(const MArgList& args)
         }
 
         // Read all of the dictionary args first.
-        const VtDictionary userArgs = UsdMayaUtil::GetDictionaryFromArgDatabase(
+        VtDictionary userArgs = UsdMayaUtil::GetDictionaryFromArgDatabase(
```

Comment:
I'm a little wary of dropping the `const` here, as I think we want to make sure we maintain a store that's guaranteed to contain the actual arguments the user passed in. More below...